liberate/cells.tcl

```diff
@@ -0,0 +1,8 @@
+# Comment
+set cells {
+  AND2X1
+  INVX1
+  XOR2X1
+}
+# Overrides
+set cells { AND2X1 INVX1 }
```

Comment:
Should there be only one `set cells { ... }` block? I assume the second overwrites the first?